### DISCUSSION TITLE
Filter for multiple keywords possible?

### DISCUSSION THREAD

**Discussion by henningBunk** [jonaswinkler/paperless-ng#783] [Support] [answered]
Hi! Is it somehow possible to filter for multiple keywords?

Use case, I bought a Samsung device on Amazon and I want to search for the invoice. Therefore I type `amazon samsung` in the filter field. Since in the invoice the strings don't stand next to each other I get zero results. I also tried: 

* amazon, kindle
* amazon and kindle
* amazon AND kindle
* "amazon", "kindle"
* amazon && kindle

Is there a syntax for such logical operations?
Thanks for you help!

**Comment by jonaswinkler** [comment, marked as answer]
The title and content filters do not have any intelligent logic behind them. They simply return documents where the filter strings appears in the title, or content, case insensitive. 

The full text search on the top has much more intelligent searching logic, including boolean expresssions, see https://paperless-ng.readthedocs.io/en/latest/usage_overview.html#searching. If multiple words are entered there, it searches for documents containing all of them by default.

I'm working on better integrating the search results in the main application, so that you'll eventually be able to also use filtering and bulk editing on its results as well.

**Reply by henningBunk** [reply]
Thanks for your quick reply. This is perfect and will serve all my needs :)

Somehow I completly missed that part of the docu. Catching up now.
